mapie/regression/regression.py

```diff
@@ -708,74 +540,22 @@ def predict(
         self._check_ensemble(ensemble)
         alpha = cast(Optional[NDArray], check_alpha(alpha))
 
-        y_pred = self.single_estimator_.predict(X)
-        n = len(self.conformity_scores_)
-
-        if alpha is None:
-            return np.array(y_pred)
```

Comment:
UPD: keep these lines.

Reply:
They are in the `else` block at lines 560-561.

Reply:
In order to keep the logic of Mapie, I suggest you to move back the `else` block from lines 560-561 to line 543 as done before.

Reply:
Done